das/aggregator.go

```diff
@@ -248,6 +265,17 @@ func (a *Aggregator) Store(ctx context.Context, message []byte, timeout uint64)
 	return &aggCert, nil
 }
 
+func (a *Aggregator) KeysetFromHash(ctx context.Context, ksHash []byte) ([]byte, error) {
+	if !bytes.Equal(ksHash, a.keysetHash[:]) {
+		return nil, ErrDasKeysetNotFound
+	}
+	return a.keysetBytes, nil
```

Comment:
We'll want to retrieve this from on-chain so we can change keysets without rolling out a node config change

Reply:
The aggregator is an independent module that can run outside of a node, and can serve multiple nodes. This API method is only supposed to return keysets that this aggregator (more generally this DAS) knows about.

Reply:
KeysetFromHash is used in bothSetMachinePreimageResolver and VerifyNonPayloadParts. We'll need to make sure that those are never used with a resolver than can't get historic certificates I think. This definitely makes me nervous